backend/index.js

```diff
@@ -10,6 +10,20 @@ app.get('/',(req,res)=>{
     res.send('Hello world');
 })
 
+app.get('/stockname/:NAME',(req, res)=>{
+    const headers = {headers:{'Authorization':process.env.QUOTE_API_KEY}}
+    let NAME = req.params.NAME
+    axios.get(`https://finnhub.io/api/v1/quote?symbol=${NAME}&token=bv6jgev48v6s9eue2ndg`, headers)
+    .then(function(response){
+        console.log(response);
+        res.status(200).json(response.date);
```

Comment:
I ran this and didnt get any data back, it might be data instead of date here.

Reply:
Fixed the issue. Originally was just looking at console for output data, now I changed date to data and now the information is being presented on the webpage.